test/idp/src/main/kotlin/org/codice/compliance/web/sso/PostLoginTest.kt

```diff
@@ -64,14 +66,17 @@ class PostLoginTest : StringSpec() {
                 }
             }.apply { SimpleSign().signSamlObject(this) }
 
-            return authnRequestToString(authnRequest)
+            val authnRequestString = authnRequestToString(authnRequest)
+            Log.debug(prettyPrintXml(authnRequestString))
```

Comment:
❓ Is there either a conditional test - `isDebugEnabled`- or a version of the methods that takes a lambda? Otherwise, we're going to perform the pretty print formatting every time, even when the log level isn't set to debug and we don't print the log message.

Reply:
There isn't an `isDebugEnabled` type of method. I can get the log level and check if it's `DEBUG`. 
Looking at other kotlin-based loggers none of them have a log enabled method. Not sure if it's the same with this logger but another one I saw explicitly mentions that you do not need to check if a log level is enabled because it does that check internally.